Added bots section and Pixel bot (#2)
workadventure/awesome-workadventure: README.md

- 🎥 [WorkAdventure - Return to Office without Returning to the Office!](https://www.youtube.com/watch?v=Yx6xSptPA5U)
- [Community install guide](https://wiki.techinc.nl/Work-Adventure/install)

## Bots

_A list of bots that can be added to your workadventure map_

> - [Pixel](https://github.com/rllola/wa-bot) - He is a bridge between the office/venue you created to others applications (e.g slack)

Reviewer: ```suggestion
- [Pixel](https://github.com/rllola/wa-bot) - Pixel is a bridge between the office/venue you created and other applications (e.g slack)
```
